flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py

```diff
@@ -67,17 +68,21 @@ def run(self) -> None:
                 logging.error("Unexpected Exception", exc_info=e)
 
     def _process_event(self, event: BaseEvent):
-        # ignore scheduler event
-        if event.namespace == 'scheduler':
-            return
-        project_name = event.namespace
-        subscribed_workflow = self._get_subscribed_workflow(event, project_name)
-
-        for workflow in subscribed_workflow:
+        subscribed_workflow = []
+        projects = self.store.list_project(sys.maxsize, 0)
+        for project_name in projects:
+            workflows = self._get_subscribed_workflow(event, project_name)
+            if len(workflows) > 0:
+                for workflow in workflows:
+                    subscribed_workflow.append((project_name, workflow))
+
+        for workflow_tuple in subscribed_workflow:
+            project_name, workflow = workflow_tuple
             self._handle_event_for_workflow(project_name, workflow, event)
 
     def _get_subscribed_workflow(self, event, project_name):
         workflows: List[WorkflowMeta] = self.store.list_workflows(project_name=project_name)
+        print(workflows)
```

Comment:
`print` is not recommanded. Maybe use debug level logging.

Reply:
removed